deploy/helm/kubecf/assets/operations/instance_groups/router.yaml

```diff
@@ -39,3 +44,14 @@
 - type: replace
   path: /instance_groups/name=router/env?/bosh/agent/settings/labels/app.kubernetes.io~1version
   value: {{ default .Chart.Version .Chart.AppVersion | quote }}
+
+# Trust the diego_instance_identity_ca certificate - trusting its CA
+# is insufficient with the cf-operator 
+- type: replace
+  path: /instance_groups/name=router/jobs/name=gorouter/properties/router/ca_certs
+  value: |
+    ((diego_instance_identity_ca.ca))
+    ((cc_tls.ca))
+    ((uaa_ssl.ca))
+    ((network_policy_server_external.ca))
+    ((diego_instance_identity_ca.certificate))
```

Comment:
Do we need to trust _both_ the CA and the cert?

Reply:
No, this is only required temporarily to work around a [bug in the operator](https://www.pivotaltracker.com/story/show/170044974).

Reply:
Then we need a TODO here.